Digital Twin Manager 1025.8.0 Changelogs for UI 1025.2.0 (#4883)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-8-0-upgrade-web-sdk-version-to-1023-97-3.md

---
date: ""

> title: "Upgrade webSDK version to 1023.97.3"

Reviewer: ```suggestion
title: "Upgrade of Web SDK version to 1023.97.3"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-3060"
version: "1025.8.0"
---

> The Digital Twin Manager application has been upgraded to the Cumulocity

Reviewer: ```suggestion
The Digital Twin Manager application has been upgraded to the {{< product-c8y-iot >}}
```

> Web SDK version 1023.97.3

Reviewer: ```suggestion
Web SDK version 1023.97.3.
```
